microsoft-edge/devtools-guide-chromium/accessibility/test-tab-enter-keys.md

```diff
@@ -0,0 +1,76 @@
+---
+description: Check for keyboard support by using the Tab and Enter keys.
+title: Check for keyboard support by using the Tab and Enter keys
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/13/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# Check for keyboard support by using the Tab and Enter keys
+
+Test the usability of a webpage for keyboard users in several ways:
+*  By using the keyboard, particularly the `Tab`, `Shift`+`Tab`, and `Enter` keys.  This approach is described in the current article.
+*  Check for keyboard support for an individual element by using the **Inspect** tool.  The Inspect tool's information overlay includes an **Accessibility** section that includes a **Keyboard-focusable** row.  
+*  Check the **Issues** report's **Accessibility** section for keyboard support issues.
+
+Not all users have a pointer or touch device, and not all users can see the web projects we create.  This is why it is important that the user interface works at least with a keyboard.  Make sure you can use the `Tab` key to move the focus to each form control on a webpage, and make sure you can use the `Enter` key to submit the form.
+
+**To check the demo page for accessibility issues by using a keyboard rather than a mouse:**
```

Comment:
```suggestion

Not all users have a pointer or touch device, and not all users can see the web projects we create.  This is why it is important that the user interface works at least with a keyboard.  Ensure you can use the `Tab` key to move the focus to each form control on a webpage, and ensure you can use the `Enter` key to submit forms.

You can test the usability of a webpage for keyboard users in several ways:
*  By using the keyboard, particularly the `Tab`, `Shift`+`Tab`, and `Enter` keys.  This approach is described in this article.
*  Check for keyboard support for an individual element by using the **Inspect** tool.  The Inspect tool's information overlay includes an **Accessibility** section that includes a **Keyboard-focusable** row.  
*  Check the **Issues** report's **Accessibility** section for keyboard support issues.

To check the demo page for accessibility issues by using a keyboard rather than a mouse, perform the following steps:
```